.github/workflows/black.yml

```diff
@@ -0,0 +1,36 @@
+name: Black
+
+on:
+  push:
+    branches:
+      - v-ghajam/CI-CD-Linter
```

Comment:
Don't forget to change to `main` branch before merging

Reply:
done